website/src/Main.elm

```diff
@@ -57,4 +67,4 @@ update action model =
 view : Address Action -> Model -> Html
 view address model =
   div [ class "wrapper" ]
-    [ Router.view address model ]
\ No newline at end of file
+    [ Signal.map3 route History.path address model ]
```

Comment:
This doesn't work, need to look into mapping the `Signal Router.Route` into `Html`